integration/base/src/main/java/com/inrupt/client/integration/base/AccessGrantScenarios.java

```diff
@@ -757,4 +755,25 @@ private static Stream<Arguments> provideSessions() throws SolidClientException {
             Arguments.of(resourceOwnerSession, requesterSession)
             );
     }
+
+    private static void createAuthenticatedClient() {
+        final Session session = OpenIdSession.ofClientCredentials(
+                URI.create(issuer), //Client credentials
+                RESOURCE_OWNER_CLIENT_ID,
+                RESOURCE_OWNER_CLIENT_SECRET,
+                AUTH_METHOD);
+
+        authResourceOwnerClient = SolidSyncClient.getClient().session(session);
+    }
+
+    private static void createContainer(final URI publicContainerURI) {
```

Comment:
Should this be considered for the public API of `SolidClient`? It seems like something  a lot of users will need.

Reply:
good point :) Will make a note about it.